pyxlsb/cell.py

```diff
@@ -13,19 +13,43 @@ def c(self):
     def v(self):
         return self.value
 
+    @property
+    def value_conv(self):
```

Comment:
Add this to the Cell class instead.

The mixin is for `1.0.x` backward compatibility only.